components/breadcrumb/docs.mdx

```diff
@@ -0,0 +1,28 @@
+---
+name: Breadcrumb
+menu: Components
+route: /breadcrumb
+---
+
+import { Playground, PropsTable } from 'docz';
+import { Breadcrumb, Crumb } from './src';
+
+# Breadcrumb
+
+Write your docs here
```

Comment:
Use `Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.` for now